webpack.config.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
         symlinks: false
   },
   output: {
-    publicPath: 'auto',
+    publicPath: '',
```

Comment:
what does this do?

Reply:
sets the base path for assets in the package. auto is used for if it's unknown where assets may come from, but in our case it's mostly defined so we probably don't need it. I think I can remove this variable altogether. https://webpack.js.org/guides/public-path/

but also, if we're moving to parcel this is likely going to get reconfigured anyways it seems.